week5_plan.md

```
@@ -0,0 +1,228 @@
+### What to build
+
+| Event | Notify who | Config flag (default) |
+|-------|------------|------------------------|
+| `pr_closed` | PR author | `notifications.pr_closed` (true) |
+| `pr_reopened` | PR author | `notifications.pr_reopened` (true) |
+| `issue_reopened` | Assignee (if any) | `notifications.issue_reopened` (true) |
+| `pr_reviewed` (COMMENT) | PR author | `notifications.pr_review_comment` (true) |
+
+### Files (expected)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)